app/routes.js

```diff
@@ -0,0 +1,25 @@
+var	express = require('express');
+var	config = require('./controller/settings');
+
+var	router = express.Router();
+var configData;
+
+// Route to CREATE a config
+router.post('/configs', function (req, res) {
+	configData = config.update(req);
+	console.log(configData);
+	res.json(configData);
+});
+
```

Comment:
You just need to do req.send(); you added your update to req object which is already getting sent